SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security policy
+
+## Supported versions
+
+Security updates will be released after vulnerabilities have been identified and fixed. At this point in time,
+we will only release security patches to the `latest/edge` and `latest/candidate` charm release channels.
```

Comment:
Same as with `filesystem-charms`, IIRC we don't currently use the `latest/candidate` channel:

```suggestion
Security updates for the Slurm charms will be released after vulnerabilities have been identified and fixed to the `latest/edge` and `latest/stable` release channels.
```

We'll eventually want to sit down after mid-cycle and identify our release strategy. E.g. what version do we track, how will we split charms into channels, etc. However, what we have now is good!